### ISSUE TITLE
[Bug]: No "create brain" options when running on WSL

### ISSUE THREAD

**Issue by pachadotdev** [QuivrHQ/quivr#2332]
### What happened?

A bug happened! I installed Quivr from the main git branch on WSL (I couldn't get to install Scoop). It runs on localhost:3000, but when I click on "create a new brain", the displayed rectangle shows 0 options. What should I do? I added the OPENAI token as OPENAI_API_KEY=111222333, should it be OPENAI_API_KEY="111222333"? I already tried with quotes.

**Comment by pachadotdev** [comment]
@dosu-bot Hi! I implemented those and it didn't work.  I used both the LAN local IP and WSL IP. The problem persists.

**Comment by Croccodoyle** [comment]
duplicate of 2322

**Comment by pachadotdev** [comment]
> duplicate of 2322

hi @Croccodoyle , I do not really understand why it is a duplicate, I just configured Quivr folowing the tutorial

**Comment by stef-bloom** [comment]
I ran into the same thing: no options to choose from when trying to create a new brain, despite running migrations. 
I was able to fix it by editing some db tables that were missing entries. 
I may have forgotte a step ( I threw a lot of spaghetti at the wall), but this might point you in the right direction.

As a side note, I occasionally run into issues with the scripts, where supabase throws an error "[bla bla] script is too long [bla bla]" or it throws some other error that stops execution mid-script. I think that's what happened here for me. Something died mid-script before getting ot the good parts.

As a sanity check, I created a brand new db with all the scripts (and ran the too long init script in two halves). 
When I compared the two dbs, the (older) one, which was not giving me brain options was missing entries in these tables:

- integrations (empty, no rows)
- notifications (empty, no rows)
- onboardings (-> make sure there is a record for the user/tester uuid that is logged in)

To fix the issue try the following:

- in supabase SQL Editor, RUN (from the seed file):
-- the "id" vals here are not related to any uuids, so they should work as is
-- ensure that the field "tags" in "integrations" is not NULL (see screenshot); if it's not an empty set (brackets), you'll get an application error

INSERT INTO "public"."integrations" ("created_at", "integration_name", "integration_logo_url", "connection_settings", "id", "description", "integration_type", "max_files", "information", "tags", "allow_model_change", "integration_display_name") VALUES
	('2024-03-06 21:21:07.251232+00', 'doc', 'https://quivr-cms.s3.eu-west-3.amazonaws.com/225911_200_46634039e4.png', NULL, 'b37a2275-61b3-460b-b4ab-94dfdf3642fb', 'Default description', 'custom', 5000, NULL, '{}', true, 'Brain');


INSERT INTO "public"."notifications" ("id", "datetime", "chat_id", "message", "action", "status") VALUES
	('e6755dde-932e-48d1-a949-4a3357ead3ca', '2024-02-21 02:17:47.998437', 'ec676cad-3c44-4da0-8090-dca235627ebb', NULL, 'UPLOAD', 'Pending');


![image](https://github.com/QuivrHQ/quivr/assets/50852842/6b72b7b5-b8b2-4629-964c-dcf9f7e9421c)
